test/e2e/__init__.py

```diff
@@ -372,3 +374,15 @@ def get_sagemaker_pipeline(pipeline_name: str):
             f"SageMaker could not find a pipeline with the name {pipeline_name}. Error {error}"
         )
         return None
+
+
+def delete_custom_resource(
+    reference,
+    wait_period=cfg.JOB_DELETE_WAIT_PERIODS,
+    wait_length=cfg.JOB_DELETE_WAIT_LENGTH,
+):
+    deleted = True
```

Comment:
nit: default to false?

Reply:
Should default to true in case where resource does not exist so that assert passes. If resource existed the deletion method is called it will assign the true or false value as it normally would.